src/main/java/com/hazelcast/commandline/managementcenter/ManagementCenterCommandLine.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.hazelcast.commandline.managementcenter;
+
+import com.hazelcast.commandline.AbstractCommandLine;
+import com.hazelcast.commandline.VersionProvider;
+import com.hazelcast.commandline.ProcessExecutor;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.hazelcast.internal.util.StringUtil.isNullOrEmpty;
+
+/**
+ * Commandline class for Hazelcast Management Center operations
+ */
+@CommandLine.Command(name = "mc", description = "Utility for Hazelcast Management Center operations.",
+        versionProvider = VersionProvider.class, mixinStandardHelpOptions = true, sortOptions = false)
+public class ManagementCenterCommandLine extends AbstractCommandLine {
+
+    private String mcVersion;
+
+    public ManagementCenterCommandLine(PrintStream out, PrintStream err, ProcessExecutor processExecutor)
+            throws IOException {
+        super(out, err, processExecutor);
+        mcVersion = new VersionProvider().getMcVersion();
+    }
+
+    @Override
+    public void run() {
+        List<CommandLine> parsed = spec.commandLine().getParseResult().asCommandLineList();
+        if (parsed != null && parsed.size() == 1) {
+            spec.commandLine().usage(out);
+        }
+    }
+
+    @CommandLine.Command(description = "Starts a new Hazelcast Management Center instance", mixinStandardHelpOptions = true)
+    public void start(
+            @CommandLine.Option(names = {"-c", "--context-path"}, paramLabel = "<context-path>",
+                    description = "Bind to the specified <context-path>.")
```

Comment:
I'm not sure how well people know what is MC context? Consider adding some explanation.

Reply:
👍